include/bluetooth/mesh/scheduler_srv.rst

```diff
@@ -0,0 +1,222 @@
+However, the Scheduler Server skips configuration parameters not allowing to calculate the exact time of the action.
+Such actions will never be executed.
+
+When the scheduled action is executed, the Scheduler Server sends a notification about the action in progress to appropriate Scene or OnOff Server instances, starting with the same element on which the Schedule Server is present.
+The Scene or OnOff Server instances are notified until the Scheduler Server reaches another element with a Scheduler Server instance, or there are just no elements left.
+The Scene or OnOff Servers publish their states if the states have changed.
+
+The application should notify the Scheduler Server if the Time Zone or the UTC time has changed, using the Scheduler Server API:
+
+* :c:func:`bt_mesh_scheduler_srv_time_update`: Notify server about UTC time or Time Zone changes
+
+The Scheduler Server will recalculate time for the actions.
+
+States
+======
+
+Schedule Register: :c:type:`bt_mesh_schedule_entry`
+   The Scheduler Server model offers a Schedule Register that contains up to sixteen registered action entries.
+
+Each entry has the following fields:
+
+* Year: 2 least significant digits of the year
+
+  +------------------+----------------------------------------+
+  | Value            | Description                            |
+  +------------------+----------------------------------------+
+  | 0x00-0x63        | 2 least significant digits of the year |
+  +------------------+----------------------------------------+
+  | 0x64             | Any year                               |
+  +------------------+----------------------------------------+
+  | All other values | Prohibited                             |
+  +------------------+----------------------------------------+
+
+* Month: Bitmask of the months in the year in which the scheduled event is enabled
+
+  +-----+------------------------+
+  | Bit | Description            |
+  +-----+------------------------+
+  | 0   | Scheduled in January   |
+  +-----+------------------------+
+  | 1   | Scheduled in February  |
+  +-----+------------------------+
+  | 2   | Scheduled in March     |
+  +-----+------------------------+
+  | 3   | Scheduled in April     |
+  +-----+------------------------+
+  | 4   | Scheduled in May       |
+  +-----+------------------------+
+  | 5   | Scheduled in June      |
+  +-----+------------------------+
+  | 6   | Scheduled in July      |
+  +-----+------------------------+
+  | 7   | Scheduled in August    |
+  +-----+------------------------+
+  | 8   | Scheduled in September |
+  +-----+------------------------+
+  | 9   | Scheduled in October   |
+  +-----+------------------------+
+  | 10  | Scheduled in November  |
+  +-----+------------------------+
+  | 11  | Scheduled in December  |
+  +-----+------------------------+
```

Comment:
Most of these tables are just duplications of the enums they map to. I think the fields that have enums should just link those, The ones that are numbers, but have special defines can still be tables here, but they should reference the defines themselves, not the raw values in the defines.